cardinal/system/t_meat_puppet_system_test.go

```diff
@@ -0,0 +1,290 @@
+	var er uint8
+	var move bool
+	var output string
+	//var nxt uint32 ---> Not used YET
+
+	// Start a new token system
+	ts = NewTokeniserSystem()
+
+	if uint8(len(tokens)) > constants.MAX_TOK {
+		er = constants.ErrParserRoutineTKCX.Code
+		output = constants.ErrParserRoutineTKCX.Message
+	} else {
+		var tok1 string
+		tok1 = tokens[0]
+		tokD := ts.GetDirectionType(tok1)
+
+		if tokD != enums.DirectionTypeNone {
+			move = true
+			// HERE GOES GET NEXT ROOM - DIRECTION SYSTEM
+		} else if ts.GetActionType(tok1) != enums.ActionTypeNone {
+			if uint8(len(tokens)) >= constants.MIN_TOK {
+
+				if ts.GetActionType(tok1) == enums.ActionTypeGo {
+					// GO: form
+					move = true
+					output = "GOING TO NEXT ROOM - DIRECTION SYSTEM - TO BE IMPLEMENTED"
+					// HERE GOES GET NEXT ROOM - DIRECTION SYSTEM
+				} else {
+					// VERB: form
+					output, er = handleVerbTest(tokens, rID, pID)
+					move = false
+				}
+
+			} else {
+				er = handleAliasTest(tokens, pID)
+				move = false
+				output = "VERB GOES TO HANDLE ALIAS. TO BE IMPLEMENTED"
+			}
+		} else {
+			er = constants.ErrParserRoutineNOP.Code
+			output = constants.ErrParserRoutineNOP.Message
+		}
+	}
+
+	return output, move, er
+}
+
+func handleAliasTest(tokens []string, playerID uint32) (err uint8) {
+	vrb := ts.GetActionType(tokens[0])
+	//var e uint8
+	if vrb == enums.ActionTypeInventory {
+		fmt.Printf("---->HANDLE ALIAS: NOW SHOULD BE GOING TO INVENTORY FROM INVENTORY SYSTEM")
+		err = 135 // This is just for showing errors
+	} else if vrb == enums.ActionTypeLook {
+		fmt.Printf("--->HANDLE ALIAS: NOW SHOULD BE GOING TO STUFF FROM LOOK SYSTEM")
+		err = 130 // This is just for showing errors
+	}
+	return err
+}
+
+// Handle if the token is a verb
+func handleVerbTest(tokens []string, roomID uint32, playerID uint32) (output string, err uint8) {
+	vrb := ts.GetActionType(tokens[0])
+	var e uint8
+	var resultStr string
+
+	switch vrb {
+	case enums.ActionTypeLook, enums.ActionTypeDescribe:
+		fmt.Printf("---->HANDLE VERB: NOW SHOULD BE GOING TO STUFF FROM LOOK SYSTEM")
+		resultStr = ("---->HANDLE VERB: NOW SHOULD BE GOING TO STUFF FROM LOOK SYSTEM")
+		e = 0
+
+	case enums.ActionTypeTake:
+		fmt.Printf("---->HANDLE VERB: NOW SHOULD BE GOING TO TAKE FROM INVENTORY SYSTEM")
+		resultStr = "---->HANDLE VERB: NOW SHOULD BE GOING TO TAKE FROM INVENTORY SYSTEM"
+		e = 0
+
+	case enums.ActionTypeDrop:
+		fmt.Printf("---->HANDLE VERB: NOW SHOULD BE GOING TO DROP FROM INVENTORY SYSTEM")
+		resultStr = "---->HANDLE VERB: NOW SHOULD BE GOING TO DROP FROM INVENTORY SYSTEM"
+		e = 0
+
+	default:
+		fmt.Printf("---->HANDLE VERB: NOW SHOULD BE GOING TO ACT FROM ACTION SYSTEM")
+		resultStr = "---->HANDLE VERB: NOW SHOULD BE GOING TO ACT FROM ACTION SYSTEM"
+		e = 0
+		fmt.Printf("---->HANDLE VERB:resultStr: %s", resultStr)
+	}
+
+	return resultStr, e
+}
+
+// returns a string to be used on the message variable of the transaction/message
+func insultMeatTest(cErr uint8, badCmd string) string {
```

Comment:
again this is not a test it returns something to a tests so its testing a test.

Reply:
Leaving this file until get response from Argus guys